.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: black
+    name: black
+    entry: black
+    language: python
+    language_version: python3.6
+    files: \.(py|gyp|gypi|tac|wsgi)$
```

Comment:
Note that [`types`](https://pre-commit.com/#filtering-files-with-types) was added recently and offers a more flexible way to do this

I think what would work best here is:

```yaml
    types: [python]
```

Note that this'll also allow it to apply to extensionless files which have the proper `python` shebang

Reply:
At the moment "python" only matches .py though:
https://github.com/chriskuehl/identify/blob/master/identify/extensions.py#L86

Reply:
Indeed, I'll work with @chriskuehl to get a few more extensions added there 👍

Reply:
but yeah, don't block on that -- go ahead and ship this as is and we can iterate in `identify` separately

Reply:
Sweet, thanks for the list :D

Reply:
Last question: why is `[python]` not enough? I thought it would implicitly assume it's a text file (per identify)?

Reply:
Oh yeah [python] would be enough. I was misremembering from  an earlier implementation where symlinks to files would be classified as the thing they point to. This isn't the case so [python] should be good!